SpywareFilter/sections/allowlist.txt

```diff
@@ -37,6 +37,8 @@
 !#endif
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/139559
+@@||aliexpress.*^$cookie=/^_ga_/
```

Comment:
How to reproduce it? I can't with `.com` and `.us`

Reply:
> How to reproduce it?

Please read `Add your comment and screenshots`.

Reply:
I get this error (even without AdGuard)

<details>
<summary>Details</summary>

![image](https://user-images.githubusercontent.com/8361299/211384576-bf922ca1-c279-4fb8-8e22-c3aa03cd18da.png)

</details>

@jellizaveta @Sergey-Lyapin please take a look.

Reply:
I have the same error and the rule above helps.